config/index.ts

```diff
@@ -8,83 +12,100 @@ const config: NetworkConfig = {
   ALCHEMY_GOERLI_KEY: process.env.ALCHEMY_GOERLI_KEY || '',
   ALCHEMY_RINKEBY_KEY: process.env.ALCHEMY_RINKEBY_KEY || '',
   ALCHEMY_ROPSTEN_KEY: process.env.ALCHEMY_ROPSTEN_KEY || '',
+  ALCHEMY_MATIC_KEY: process.env.ALCHEMY_MATIC_KEY || '',
+  ALCHEMY_MUMBAI_KEY: process.env.ALCHEMY_MUMBAI_KEY || '',
+
   ETHERSCAN_API_KEY: process.env.ETHERSCAN_API_KEY || '',
 
   MAINNET_PRIVATE_KEY: process.env.MAINNET_PRIVATE_KEY || '',
   GOERLI_PRIVATE_KEY: process.env.GOERLI_PRIVATE_KEY || '',
   RINKEBY_PRIVATE_KEY: process.env.RINKEBY_PRIVATE_KEY || '',
   ROPSTEN_PRIVATE_KEY: process.env.ROPSTEN_PRIVATE_KEY || '',
+  MATIC_PRIVATE_KEY: process.env.MATIC_PRIVATE_KEY || '',
+  MATIC_MUMBAI_PRIVATE_KEY: process.env.MATIC_MUMBAI_PRIVATE_KEY || '',
 
   CHAINLINK_MAINNET_PRICE_FEED:
     process.env.CHAINLINK_MAINNET_PRICE_FEED || '0x0000000000000000000000000000000000000000',
-  CHAINLINK_KOVAN_PRICE_FEED: process.env.CHAINLINK_KOVAN_PRICE_FEED || '0x0000000000000000000000000000000000000000',
```

Comment:
i removed all the kovan and goerli stuff cause those networks are not usable with our configuration